tree/ntuple/v7/inc/ROOT/RColumn.hxx

```diff
@@ -75,8 +75,15 @@ private:
    NTupleSize_t fFirstElementIndex = 0;
    /// Used to pack and unpack pages on writing/reading
    std::unique_ptr<RColumnElementBase> fElement;
+   /// The column team is a set of columns that serve the same column index for different representation IDs
+   /// Initially, the team has only one member, the very column it belongs to. Through MergeTeams(), two column
+   /// can join forces. The team is used to react on suppressed columns: if the current team member has a suppressed
+   /// column for a MapPage() call, it get the page from the active column in the corresponding cluster.
+   std::shared_ptr<std::vector<RColumn *>> fTeam;
```

Comment:
I wonder if we expect this to be shared by enough columns to justify using a shared_ptr rather than replicating the vector in each column. In my understanding the number of owners should be equal to the number of different representations for the column, right? If this is going to be 1 in the majority of cases, maybe each column could only store every column except itself, leading to an empty vector (and a saved allocation) in most cases.

Reply:
I removed the shared pointer but I kept `this` part of the vector because I find that otherwise it complicates the switching logic in `TryMapPage` too much for the gain.